test/functional/fixtures/regression/gh-423/test.js

```diff
@@ -0,0 +1,25 @@
+describe('[Regression](GH-423)', function () {
+    it('Should raise click event except in Firefox if target element appends child after mousedown', function () {
+        return runTests('testcafe-fixtures/index.test.js', 'Raise click if target appends child', { skip: ['firefox', 'firefox-osx'] });
```

Comment:
hmmm, yes. My mistake. But on my feeling it should be conversely. 
I see no cases when the `skipOption.indexOf(browserInfo.settings.alias)` check may be useful, but now we have a case when the `browserInfo.settings.alias.indexOf(skipOption)` is better